scripts/lint.sh

```diff
@@ -4,4 +4,4 @@ set -e # exit when error
 
 printf "\nLint\n"
 
-eslint . --quiet
+eslint . --quiet --no-color
```

Comment:
I am ok with keeping it if it simplifies dev workflow for some developers. No big deal